_categories/models_links.md

```diff
@@ -29,10 +29,10 @@ subcategories:
         desc: 'Link to the official sdformat API'
         star: false
         
-      - title: 'Generating multiple identical objects'
-        url: 'http://sdformat.org/api' # check
-        desc: 'How you can create a population of models by using the SDF'
-        star: false
+      # - title: 'Generating multiple identical objects'
```

Comment:
@j-rivero 

I have copied all these entries from the minimal stub you implemented.
I think you intended another url for this entry.

Reply:
right, not an easy one to found even using google knowing that there is a tutorial for this. It took me two minutes: http://gazebosim.org/tutorials?tut=model_population&cat=build_world

Reply:
Haha, thanks